supabase/supabase
# fix: column primary key toggling errors

## What kind of change does this PR introduce?

Fixes https://github.com/supabase/supabase/issues/13651

## What is the current behavior?

Error when edit column with name containing upper case character

## What is the new behavior?

Toggling primary key config for columns works without any errors

## Steps to test

Create a table with a column contains name with an upper case letter

- Go to table
- Click on edit column
- Toggle primary key config for this column
- Make sure all actions are done without any errors

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.
